test/Microsoft.AspNetCore.Mvc.Core.Test/Internal/ControllerActionInvokerCacheTest.cs

```diff
@@ -186,8 +186,80 @@ public void GetControllerActionMethodExecutor_CachesExecutor()
             Assert.Same(executor1, executor2);
         }
 
+        [Theory]
```

Comment:
Right, but this is a unit test for a method that isn't called by product code :trollface: